Sources/ComposableArchitecture/SwiftUI/TextState.swift

```diff
@@ -239,14 +247,20 @@ extension Text {
         return text.tracking(tracking)
       case let .underline(active, color):
         return text.underline(active, color: color)
+      case .multilineTextAlignment:
+        return text // applied in view body
       }
     }
   }
 }
 
 extension TextState: View {
   public var body: some View {
-    Text(self)
+    if let alignment = modifiers.compactMap(/Modifier.multilineTextAlignment).first {
```

Comment:
```suggestion
    if let alignment = modifiers.compactMap(/Modifier.multilineTextAlignment).last {
```
In order to be homogenous with the way modifiers are applied when comparing states (via `reduce`, where the last one of a kind wins), I think this should be `last` here.